orchestration/src/main/java/com/sap/ai/sdk/orchestration/ConfigToRequestTransformer.java

```
@@ -113,4 +123,45 @@ static InnerModuleConfigs toModuleConfigs(@Nonnull final OrchestrationModuleConf
 
     return OrchestrationConfigModules.createInnerModuleConfigs(moduleConfig);
   }
+
+  @Nonnull
+  static CompletionPostRequest fromReferenceToCompletionPostRequest(
+      @Nullable final OrchestrationPrompt prompt,
+      @Nonnull final OrchestrationConfigReference reference) {
+    List<ChatMessage> messageHistory = List.of();
+    Map<String, String> placeholders = Map.of();
+    if (prompt != null) {
+      if (!prompt.getMessages().isEmpty()) {
+        log.debug(
+            "Messages in prompts are ignored when using Orchestration configs via reference. Change the Orchestration config instead.");
+      }
```

This could also throw instead of just logging. Was not sure which is better.